tor/core/inbox.py

```diff
@@ -31,6 +32,16 @@ def check_inbox(config):
     replies = []
     # grab all of our messages and filter
     for item in config.r.inbox.unread(limit=None):
+        # Very rarely we may actually get a message from Reddit itself.
+        # In this case, there will be no author attribute.
+        if item.author is None:
+            send_to_slack(
+                'We received a message without an author. Subject: {}'
+                ''.format(item.subject), config
```

Comment:
Wait, does this work with python? I thought `format()` would act on the empty string, then concatenate the two strings that are (relatively) side-by-side.

Reply:
<img width="150" alt="screenshot 2017-09-20 14 57 10" src="https://user-images.githubusercontent.com/5179553/30662131-038160d4-9e14-11e7-9f1e-2202b304a9de.png">

Reply:
Cool! I figured you knew what you were doing there. The python strings-next-to-each-other-are-concatenated concept breaks my brain.